cat-theme/src/css/content/_large-cards.scss

```diff
@@ -0,0 +1,93 @@
+.large-cards {
+    &--wrapper {
+        margin-bottom: 7rem;
+    }
+    
+    &--container {
+        padding: 0;
+        width: 100%;
+    }
+}
+
+.large-card {
```

Comment:
What is the difference between `large-cards` and `large-card`?  Since there is only 1 letter difference, we should try to make the CSS class more explicit or different so at a quick glance they do not look identical.

Ex.
```
.large-card-wrapper {...}

.large-card-content {...}
```